apps/server/src/modules/user/uc/user.uc.ts

```diff
@@ -22,7 +21,7 @@ export class UserUC {
 	}
 
 	private checkAvaibleLanguages(settedLanguage: LanguageType): void | Error {
-		if (!this.userConfig.getAvailableLanguages().includes(settedLanguage)) {
+		if (!this.configService.get<string[]>('AVAILABLE_LANGUAGES').includes(settedLanguage)) {
```

Comment:
For the moment they want the type like <string[]> again. But i do not found the reasons. Because we say by set explicit which type it is. We also have activate the type check over infer. We put the interface to this subset that include the type. We say yes it is already validated and all is find. Can you please spend max 30 min to figure out if we can avoid the additional <string[]>?

Reply:
We can check it in next iteration(ticket: BC-1671)